src/azure-cli/azure/cli/command_modules/keyvault/_help.py

```diff
@@ -564,10 +568,14 @@
 type: command
 short-summary: Update the properties of a Vault.
 examples:
-  - name: Update the properties of a Vault. (autogenerated)
+  - name: Update the properties of a key vault. (autogenerated)
     text: |
         az keyvault update --enabled-for-disk-encryption true --name MyKeyVault --resource-group MyResourceGroup
     crafted: true
+  - name: Update the properties of a Vault. (autogenerated)
+    text: |
+        az keyvault update --enable-soft-delete true --name MyKeyVault
```

Comment:
pls remove example with `--enable-soft-delete`, this property is deprecating

Reply:
removed